JShellAPI/README.MD

```diff
@@ -87,18 +87,28 @@ The following startup scripts id can be used :
 - EMPTY : no startup script
 - CUSTOM_DEFAULT : contains basic imports, print methods and range method
 ## Configuration
+
+### Properties
 Properties can be defined in resources/application.properties
-### jshellapi.regularSessionTimeoutSeconds
+#### jshellapi.regularSessionTimeoutSeconds
 The timeout of a regular session, in seconds, see [Session timeout](#Session-timeout).
-### jshellapi.oneTimeSessionTimeoutSeconds
+#### jshellapi.oneTimeSessionTimeoutSeconds
 The timeout of a one-time session, in seconds, see [One-time session timeout](#One-time-session-timeout).
-### jshellapi.evalTimeoutSeconds
+#### jshellapi.evalTimeoutSeconds
 The timeout of an evaluation, in seconds, see [Eval timeout](#Eval-timeout)
-### jshellapi.maxAliveSessions
+#### jshellapi.maxAliveSessions
 The maximum number of alive sessions, see [Errors](#Errors)
-### jshellapi.dockerMaxRamMegaBytes
+#### jshellapi.dockerMaxRamMegaBytes
 The maximum ram allocated per container, in megabytes.
-### jshellapi.dockerCPUsUsage
+#### jshellapi.dockerCPUsUsage
 The cpu configuration of each container, see [--cpus option of docker](https://docs.docker.com/config/containers/resource_constraints/#cpu).
-### jshellapi.schedulerSessionKillScanRate
-The rate at which the session killer will check and delete session, in seconds, see [Session timeout](#Session-timeout).
\ No newline at end of file
+#### jshellapi.schedulerSessionKillScanRate
+The rate at which the session killer will check and delete session, in seconds, see [Session timeout](#Session-timeout).
+
+### Testing
+**JShellWrapper** Docker image lifecycle is handled during tests.
+
+The image name is injected from the root [build.gradle](../build.gradle) file, to this project's [build.gradle](build.gradle) file and also to [application.yaml](src/main/resources/application.yaml).
+
+- JShellWrapper Docker Image is built before the tests run.
+- **Container & Cleanup**: After the tests finish, the container and image are removed to ensure a clean environment.
```

Comment:
Like I said last time, this is the wrong place for this, this is the readme, not the doc.
But, you didn't mention how to run the tests, which is important to have in the readme. And if you add it, prefer adding it to the readme in the parent project, JShellPlaygroundBackend + how to run JShellWrapper if there is no mention of it.